### DISCUSSION TITLE
nilup is downloading and installing the SDK twice

### DISCUSSION THREAD

**Discussion by a3s7p** [NillionNetwork/discussions#125] [Bugs]
If following the [current docs](https://docs.nillion.com/quickstart-install) on macOS Sequoia on M1, `nilup` appears to download/install the SDK archive twice.

It is aware that it is performing a reinstall.

```
asp@huge-dog ~ % nilup -V
nilup 86bb6bd5262c9a223f0895670bba2bfe37ca02ef
asp@huge-dog ~ % nilup install latest
Installing SDK bins version latest
Downloading latest/nillion-sdk-bins-aarch64-apple-darwin.dmg
SDK version latest installed
asp@huge-dog ~ % nilup use latest
SDK version latest set as default
asp@huge-dog ~ % nilup init
SDK version latest already installed
Reinstalling SDK version latest
Installing SDK bins version latest
Downloading latest/nillion-sdk-bins-aarch64-apple-darwin.dmg
SDK version latest installed
SDK version latest set as default
```

**Comment by oceans404** [comment]
Thanks for reporting!

**Reply by jwolski2** [reply]
Piling on what @oceans404 said: thanks for reporting, @a3s7p. Indeed our doc's instructions are a bit redundant. We're making a change to the install script and the docs to make initial installation of nilup and the SDK easier. Hang tight for the release.

**Comment by jwolski2** [comment]
The new release of the `install.sh` script has been published and documentation has been updated to reflect the new behavior: https://docs.nillion.com/quickstart-install.
